### ISSUE TITLE
We looked but did not find a synpress.json file in this folder

### ISSUE THREAD

**Issue by ChrisChan888** [Synthetixio/synpress#616]
This is my repo, https://github.com/ChrisChan888/try-synpress
I did have synpress.json in the root folder, but after npx synpress run.
This error comes out.
```
Could not find a Cypress configuration file.

We looked but did not find a synpress.json file in this folder: /.../try-cypress
Failed to run Cypress
Could not find Cypress test run results
```

**Comment by reasonmethis** [comment]
I have the same issue. I of course double checked that the name is correct and located exactly in the folder where it says it didn't find it.

**Comment by jlavasyou** [comment]
I think this is due to synpress just throwing a default filesystem error from Cypress and not handling specifics

try:

`npx cypress open --config-file=synpress.json`

And it will give you more information about why it can't read the file. For me it was a bad json configuration

**Comment by ComradeAERGO** [comment]
When I use
`"env-cmd -f .env npx cypress open --config-file=synpress.json --config supportFile='cypress/support/index.js'"`
Cypress launches correctly
But when I try with 
"env-cmd -f .env npx synpress open --config-file=synpress.json --config supportFile='cypress/support/index.js'"
I get this error
```Could not find a Cypress configuration file.

We looked but did not find a synpress.json file in this folder: /Users/aerg/works/folder
[1]    60066 terminated  npm run test:e2e`

**Comment by Ed-Marcavage** [comment]
Did anyone figure this out? Im still stuck on this

**Comment by defiled** [comment]
Yes, this is very annoying. The issue stems from an update on Cypress's end in which they move away from `.json` to `.js` for their configuration files. So if you change the extention and ensure it aligns with these [specifications](https://docs.cypress.io/guides/references/configuration) it should work. 
However, you'll run into other issues so the best thing to do is just not use a config file and configure everything to work with Synpress's default configuration.
